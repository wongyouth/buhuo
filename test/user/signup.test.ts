import { get, post } from '../test_helper'

describe('用户注册', () => {
  it('foo', async () => {
    const res = await get('/hello')

    expect(res).toMatchSnapshot()
  })

  it('signup', async () => {
    const res = await post('/signup', {
      name: 'Name',
      mobile: '18012341234',
      password: 'password',
    })

    expect(res).toMatchSnapshot()
  })
})
