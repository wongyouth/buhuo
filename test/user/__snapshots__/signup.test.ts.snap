// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`用户注册 foo 1`] = `
Object {
  "data": "world",
  "status": 200,
}
`;

exports[`用户注册 signup 1`] = `
Object {
  "data": Object {
    "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjEsImlhdCI6MTYyMjIyNjMxMSwiZXhwIjoxNjMwMDAyMzExfQ.ecsRz_LWzKCLyk6DKgolLvfsSgMETdDyIp_MwYmcUUA",
    "userId": 1,
  },
  "status": 200,
}
`;
